src/opencl_aes.h

```diff
@@ -301,4 +301,48 @@ inline void AES_256_XTS_first_sector(AES_SRC_TYPE uint *in,
 		out[i] = buf[i] ^ tweak[i];
 }
 
+#define N_WORDS (AES_BLOCK_SIZE / sizeof(unsigned long))
+
+inline void
+AES_ige_decrypt(AES_CTS_SRC_TYPE void *_in, AES_CTS_DST_TYPE void *_out,
+                uint length, AES_KEY *akey, uchar *_iv)
+{
+	AES_CTS_SRC_TYPE uchar *in = _in;
+	AES_CTS_DST_TYPE uchar *out = _out;
```

Comment:
Please use `AES_SRC_TYPE` and `AES_DST_TYPE`. The more specific `AES_CTS_SRC_TYPE` was needed for some format that had different requirements at different stages.